kolibri/plugins/coach/assets/src/views/plan/assignments/IndividualLearnerSelector.vue

```diff
@@ -91,44 +112,72 @@
       selectedGroupIds: {
         type: Array,
         required: true,
+        default: new Array(),
+      },
+      entireClassIsSelected: {
+        type: Boolean,
+        required: true,
+        default: false,
+      },
+      disabled: {
+        type: Boolean,
+        required: true,
+        default: false,
+      },
+      initialIndividualLearners: {
+        type: Array,
+        required: true,
       },
     },
     data() {
       return {
-        showUserTable: false,
-        selectedIndividualIds: [],
-        selectAllOnPage: false,
+        isChecked: !!this.initialIndividualLearners.length,
```

Comment:
```suggestion
        isChecked: Boolean(this.initialIndividualLearners.length),
```

Back in #3224 we standardized on this - general philosophy is that being explicit and improving readability is better than saving a few characters